src/DssProxyActions.sol

```diff
@@ -888,22 +1196,34 @@ contract DssProxyActionsEnd is Common {
         }
         // Approves the manager to transfer the position to proxy's address in the vat
         if (vat.can(address(this), address(manager)) == 0) {
-            vat.hope(manager);
+            vat.hope(address(manager));
         }
         // Transfers position from CDP to the proxy address
-        ManagerLike(manager).quit(cdp, address(this));
+        manager.quit(cdp, address(this));
         // Frees the position and recovers the collateral in the vat registry
         EndLike(end).free(ilk);
     }
 
     // Public functions
+
+    // Deprecated
     function freeETH(
-        address manager,
+        address manager_,
         address ethJoin,
         address end,
         uint256 cdp
     ) public {
-        uint256 wad = _free(manager, end, cdp);
+        require(manager_ == address(manager), "DssProxyActions/Non-matching-manager");
```

Comment:
Should probaby update the revert messages contract to DssProxyActionsEnd.
Same in similar places below.

Reply:
Not done as I've removed the deprecated functions.